string agent_id
string current_node
string current_edge
string colour

#    string current_node
#        WayPoint140
#    string current_edge
#        WayPoint140_WayPoint141
